Assets/SEE/UI/Window/MetricWindow.cs

```diff
@@ -0,0 +1,215 @@
+        private static TextMeshProUGUI Value(GameObject metricRow)
+        {
+            return metricRow.transform.GetChild(1).GetComponent<TextMeshProUGUI>();
+        }
+
+        /// <summary>
+        /// The GameObject representing the metric window. It must be stored in this field;
+        /// otherwise, it will be garbage collected.
+        /// </summary>
+        private GameObject metricWindowObject;
+
+        /// <summary>
+        /// Creates the Metric Window
+        /// </summary>
+        public void CreateUIInstance()
+        {
+            // Instantiate MetricWindow
+            metricWindowObject = PrefabInstantiator.InstantiatePrefab(WindowPrefab, Window.transform.Find("Content"), false);
+            metricWindowObject.name = "Scrollable";
+
+            Transform scrollViewContent = metricWindowObject.transform.Find("Content/Items").transform;
+            TMP_InputField inputField = metricWindowObject.transform.Find("Search/SearchField").gameObject.MustGetComponent<TMP_InputField>();
+
+            inputField.onSelect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = false);
+            inputField.onDeselect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = true);
+
+            // Int Attributes
+            DisplayAttributes(GraphElement.IntAttributes, metricWindowObject);
+
+            // Float Attributes
+            DisplayAttributes(GraphElement.FloatAttributes, metricWindowObject);
+
+            // Save GameObjects in Array for SearchField
+            Dictionary<string, GameObject> activeElements = new();
+            foreach (Transform child in scrollViewContent)
+            {
+                activeElements.Add(AttributeName(child.gameObject), child.gameObject);
+            }
+
+            inputField.onValueChanged.AddListener(str => InputSearchField(str, activeElements));
+        }
+
+        /// <summary>
+        /// Displays the attributes and their corresponding values.
+        /// </summary>
+        /// <typeparam name="T">The type of the attribute values.</typeparam>
+        /// <param name="attributes">A dictionary containing attribute names (keys) and their corresponding values (values).</param>
+        /// <param name="metricWindowObject">The GameObject representing the metric window.</param>
+        private static void DisplayAttributes<T>(Dictionary<string, T> attributes, GameObject metricWindowObject)
+        {
+            Transform scrollViewContent = metricWindowObject.transform.Find("Content/Items").transform;
+            foreach (KeyValuePair<string, T> kvp in attributes)
+            {
+                // Create GameObject
+                GameObject metricRow = PrefabInstantiator.InstantiatePrefab(ItemPrefab, scrollViewContent, false);
+                // Attribute Name
+                Attribute(metricRow).text = kvp.Key;
+                // Value Name
+                Value(metricRow).text = kvp.Value.ToString();
+            }
```

Comment:
Note that you can deconstruct KeyValuePairs like this to get slightly more legible code:
```suggestion
            foreach ((string name, T value) in attributes)
            {
                // Create GameObject
                GameObject metricRow = PrefabInstantiator.InstantiatePrefab(ItemPrefab, scrollViewContent, false);
                // Attribute Name
                Attribute(metricRow).text = name;
                // Value Name
                Value(metricRow).text = value.ToString();
            }
```